src/tests/functional/plugin/gpu/shared_tests_instances/multi/gpu_remote_blob_tests.cpp

```diff
@@ -64,6 +64,7 @@ TEST_P(MultiDevice_Test, cannotInferRemoteBlobIfNotInitializedForDevice) {
 }
 
 TEST_P(MultiDevice_Bind_oversubsciption_test, oversubsciptionOfInferRequest) {
+    GTEST_SKIP();
```

Comment:
why?

Reply:
this can be fixed by integrating the bind PR, multi + bind, currently this part of code is missing @wgzintel

Reply:
We don't use hardcoded `GTEST_SKIP`!

Reply:
Updated